io.vikunja.Vikunja.appdata.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+	<id>io.vikunja.Vikunja</id>
+	<name>Vikunja</name>
+	<summary>The open-source, self-hostable to-do app. Organize everything, on all platforms.</summary>
+	<description>
+		<p>Think of Vikunja like the notebook you have with all your things to keep track of.</p>
+		<p>But with superpowers.</p>
+		
+		<p>☑️ Stay organized</p>
+		
+		<p>Organize all of your tasks in projects.</p>
+		<p>Create subprojects organize everything hierarchical and keep related stuff grouped together.</p>
+		
+		<p>👥 Collaborate with peers</p>
+		
+		<p>Vikunja lets you easily share a project with another user or a whole team!</p>
+		<p>Assignments show who's working on what.</p>
+		
+		<p>🧩 Use it how you need it</p>
+		
+		<p>You can view your tasks in the classic list view -or in a Gantt Chart, or Table view, or Kanban Board!</p>
+	</description>
+	<screenshots>
+		<screenshot>
+			<caption>The "Overview" page of Vikunja with a light theme</caption>
+			<image>https://kolaente.dev/vikunja/blog/media/branch/main/static/images/0.17.0-frontend.png</image>
```

Comment:
Not quite sure about hotlinking but I think we should use https://vikunja.io/images/vikunja/01-task-overview.png here as it's more up to date